optaplanner-core/src/main/java/org/optaplanner/core/impl/domain/common/ReflectionHelper.java

```diff
@@ -114,7 +114,7 @@ public static Method getGetterMethod(Class containingClass, String propertyName)
                 + (propertyName.isEmpty() ? "" : propertyName.substring(0, 1).toUpperCase() + propertyName.substring(1));
         try {
             return containingClass.getMethod(getterName);
-        } catch (NoSuchMethodException e) {}
+        } catch (NoSuchMethodException e) { }
```

Comment:
Meh. False negatives are very annoying. From practical experience, I believe that any rule which has false negatives needs to be disabled entirely - or they endanger developers from disliking CheckStyle entirely. I've seen this in practice with the maven-enforcer-plugin and Jenkins too. The messenger of bad news (= Jenkins, CheckStyle, enforcer) cannot afford to make mistakes because a developer team working against a deadlines under stress is more than willing to shoot it (= to disable it or ignore it entirely).